lib/resources/csv.rb

```diff
@@ -1,13 +1,13 @@
 # encoding: utf-8
-# author: Christoph Hartmann
-# author: Dominik Richter
 
 # Parses a csv document
 # This implementation was inspired by a blog post
 # @see http://technicalpickles.com/posts/parsing-csv-with-ruby
 module Inspec::Resources
   class CsvConfig < JsonConfig
     name 'csv'
+    supports platform: 'unix'
```

Comment:
We need to be careful, since unix and windows is not covering all operating systems. Even we have esx and cisco not being part of unix right now. While thinking about it, we may should fix that in our train config...

Reply:
For now lets add:
`platform: 'esx'`
`platform: 'cisco'`

To csv, json, yaml, and simple config. Once the OS changes go out here https://github.com/chef/train/pull/261 we can change them to 'os'. But I don't want to force out a detect change this close to 2.0 release.

Reply:
I've added this to the PR.